jobs/aws/eks-distro/attribution-files-periodics.yaml

```diff
@@ -0,0 +1,71 @@
+# Copyright 2020 Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+periodics:
+- name: attribution-files-periodic
+  # Runs every weekday (M-F) at 10am PST
+  cron: "0 18 * * 1-5"
+  cluster: "prow-postsubmits-cluster"
+  decoration_config:
+    gcs_configuration:
+      bucket: s3://prowdataclusterstack-316434458-prowbucket7c73355c-1n9f9v93wpjcm
+      path_strategy: explicit
+    s3_credentials_secret: s3-credentials
+  extra_refs:
+  - org: eks-distro-pr-bot
+    repo: eks-distro
+    base_ref: main
+  spec:
+    serviceaccountName: periodics-build-account
+    automountServiceAccountToken: false
+    containers:
+    - name: build-container
+      image: 316434458148.dkr.ecr.us-west-2.amazonaws.com/eks-distro/builder:e98fcde28fd0d85949047083e24c1daf6f961237
+      command:
+      - bash
+      - -c
+      - >
+        make update-attribution-files -C eks-distro
+        &&
```

Comment:
We don't need this part as well, as it's pertaining to image builds. Just the `make` command will suffice.

Reply:
fixed!